### ISSUE TITLE
Exoframe not updating code during subsequent deployments

### ISSUE THREAD

**Issue by rvenugopal** [exoframejs/exoframe#264]
Hello

I am trying out `exoframe` and while the deployment succeeds the first time, subsequent deployments do not appear to update the code inside my docker container for my web api, 

Commands that I  have tried 
a) `exoframe -u`
b) exoframe
c) Destroying project and re-creating.  

None of them worked

Thanks
-Rv
_________________________________________________
Exoframe version: 5.2.0
Environment : OSX Mojave
Server : Vagrant VM
App : simple nodejs app with a docker-compose.yml

**Comment by yamalight** [comment]
@rvenugopal could you please share the project you are deploying?

**Comment by rvenugopal** [comment]
@yamalight As I retried to reproduce the issue, I realized it was not an issue with exoframe.  I had 2 issues

a) I was not using `exoframe -u` for subsequent deployments
b) I realized that the  `docker-compose` file has to be different between local development and for production cause I wanted the volume to be mapped to my current folder.  However, the volume mapping did not work for the `production` version cause the current folder was not mapped to a `synced folder` on my vagrant box.

Hence closing this issue.  Thanks for your quick response
